extensions/db/src/test/java/org/mule/extension/db/integration/AbstractDbIntegrationTestCase.java

```diff
@@ -58,8 +58,7 @@
 import org.junit.runners.Parameterized;
 
 @RunnerDelegateTo(Parameterized.class)
-@ArtifactClassLoaderRunnerConfig(
-    exportClasses = {DbConnectionProvider.class, DerbyTestStoredProcedure.class})
+@ArtifactClassLoaderRunnerConfig(exportPluginClasses = {DbConnectionProvider.class, DerbyTestStoredProcedure.class})
 public abstract class AbstractDbIntegrationTestCase extends MuleArtifactFunctionalTestCase {
```

Comment:
Why is exporting DerbyTestStoredProcedure? That class should be a test class and must be available on the test without any extra change.